jinja2cli/cli.py

```diff
@@ -146,6 +150,23 @@ def _load_xml():
     return xmltodict.parse, xml.parsers.expat.ExpatError, MalformedXML
 
 
+def _load_env():
+    def _parse_env(data):
+        """
+        Parse an envfile format of key=value pairs that are newline separated
+        """
+        dict_ = {}
+        for line in data.splitlines():
+            line = line.strip()
```

Comment:
I also changed this to `lstrip` since it should only be stripping preceding spaces, not trailing spaces.